config/ldap/ldif/50_acls.ldif

```diff
@@ -0,0 +1,15 @@
+# OpenCloud ldap acl file
+dn: olcDatabase={2}mdb,cn=config
+changetype: modify
+replace: olcAccess
+olcAccess: {0}to dn.subtree="dc=opencloud,dc=eu" attrs=entry,uid,objectClass,entryUUID
+  by * read
```

Comment:
Hm, this one is tricky. I think it only works because by default we are doing every write operation as the `olcRootDN` (which regardless of any URL always has full access to everything).

What is that rule needed for? 

In OpenLDAP the first match rule always wins.